spec/support/coronavirus_feature_steps.rb

```diff
@@ -371,23 +371,13 @@ def then_the_reordered_subsections_are_sent_to_publishing_api
 
   assert_publishing_api_put_content(
     CoronavirusPage.topic_page.first.content_id,
-    request_json_includes(
-      "details" => {
-        "header_section" => "header_section",
-        "announcements_label" => "announcements_label",
-        "announcements" => [],
-        "nhs_banner" => "nhs_banner",
-        "sections_heading" => "sections_heading",
-        "topic_section" => "topic_section",
-        "live_stream" => {
-          "video_url" => LiveStream.first.url,
-          "date" => LiveStream.first.formatted_stream_date,
-        },
-        "notifications" => "notifications",
+    lambda do |request|
```

Comment:
I didn't realise `request_json_includes` was a [gds-api-adapters test helper method](https://github.com/alphagov/gds-api-adapters/blob/8c5224f9c331cf6a22d6065ef7f5de9fcbbbed20/lib/gds_api/test_helpers/publishing_api.rb#L281)!

Reply:
Me neither, it took me ages to find where it was defined. I actually ended up using `method(:request_json_includes).source_location` to find it myself.